__init__.py

```diff
@@ -0,0 +1,5 @@
+# File: __init__.py
+# Copyright (c) 2020-2021 Splunk Inc.
```

Comment:
Update Copyright year to `2021` for all the files.

Reply:
Updated.